pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/base/PensjonLatex.kt

```diff
@@ -455,9 +455,16 @@ object PensjonLatex : BaseTemplate() {
                 printWriter.printCmd("newline")
             is Element.Table ->
                 with(printWriter) {
+                    val rows = element.rows.filter {
+                        it.condition?.eval(scope) ?: true
```

Comment:
Jeg lurer litt på om `it.condition == null || it.condition.eval(scope)` leser litt bedre. Lettere å forstå at en rad skal inkluderes om den ikke har condition enn med `?: true` tenker jeg. Hva tenker du?